jaraco/text/__init__.py

```diff
@@ -637,7 +648,8 @@ def lines_from(input):
     """
     Generate lines from a :class:`importlib.resources.abc.Traversable` path.
 
-    >>> lines = lines_from(files(__name__).joinpath('Lorem ipsum.txt'))
+    >>> # from importlib_metadata import files as _files
```

Comment:
I added this comment here to help clarifying to the end-user reading the docstring where is the `files` function coming from.